Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -184,34 +318,56 @@ void HiresTexture::Prefetch()
       }
       if (iter != s_textureCache.end())
       {
-        for (const Level& l : iter->second->m_levels)
+        for (const Level& tex : iter->second->m_levels)
```

Comment:
> Actually I asked about that and they told me that the `if (base_filename.find("_mip") == std::string::npos)` line in which the data size iterator is sitting it, means everything else except the _mip files. I'm not sure yet what it does if there is a _mip, if it even counts those.

The `Load` function that's called here will load the mipmap files: https://github.com/ZexaronS/dolphin/blob/f82380b7107195983909592168b0ace3e237e05d/Source/Core/VideoCommon/HiresTextures.cpp#L311

> Otherwise this wouldn't work, since there's no _mip files in the Hypatia's Texture Pack for Zelda WW.

A texture that isn't using mipmaps is considered by Dolphin to have exactly one mipmap level.